pkg/client/testclient/fixture.go

```diff
@@ -216,3 +218,46 @@ func (o objects) Add(obj runtime.Object) error {
 
 	return nil
 }
+
+func (o objects) Delete(obj runtime.Object) error {
```

Comment:
How does this break clients or other tests?  I don't understand.  Tests must call Delete specifically.

For my test, I would like to have a PV and PVClaim for my test, ensure they bind correctly, then delete the PVClaim to test that the volume releases.  Because I use the client in my binder, removing the claim from the test client is the easiest and most natural way to mimic the user deleting the claim.